SamchukVitalii/HW7/7_3.py

```diff
@@ -0,0 +1,11 @@
+"""
+Write a function that calculates the number of characters
+included in a given string
+"""
+
```

Comment:
good job
look at the more optimized version of code

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result